guides/common/modules/ref_port-and-firewall-requirements.adoc

```diff
@@ -93,6 +93,7 @@ ifdef::satellite[]
 ifeval::["{mode}" == "connected"]
 | 443 | TCP | HTTPS | console.redhat.com | Red{nbsp}Hat Cloud plugin API calls |
 | 443 | TCP | HTTPS | cdn.redhat.com | Content Sync | https://access.redhat.com/articles/1525183[Red{nbsp}Hat CDN]
+| 443 | TCP | HTTPS | cert.console.redhat.com | Red{nbsp}Hat Insights | When using Insights, required for Inventory upload and Cloud Connector connection
```

Comment:
This should do the trick for the direct communication.
@evgeni Do you happen to know what are the ports we need for RHC?

Reply:
No, but I'd guess either 1883/8883 for mqtt or 443 for websockets/mqtt, no idea what's actually used tho.

Reply:
Added entry for `connect.cloud.redhat.com`.